### PR TITLE
ci: disable WireGuard testing in multicluster workflow

### PR DESCRIPTION
**Pull request by nbusseneau**
Issue: https://github.com/cilium/cilium/issues/18699

As discussed during community meeting on 2022-02-02, we are disabling WireGuard testing on the multicluster conformance tests workflow until someone can look into it.

### CODE REVIEW

**Comment by nbusseneau** [comment]
Link to workflow run validating the changes (WireGuard steps were skipped): https://github.com/cilium/cilium/actions/runs/1790861628

Removing temporary test commit and marking as ready for review.

**Review by christarazi — APPROVED** [review]

**Review by gandro — APPROVED** [review]
Thanks!

**Comment by nbusseneau** [comment]
This is a trivial change that does not require `github-sec`'s approval, other reviews are in, CI has passed, marking as ready to merge.

**Review by aanm — APPROVED** [review]

**Comment by nbusseneau** [comment]
After merge, we can re-enable the [multicluster workflow](https://github.com/cilium/cilium/actions/workflows/conformance-multicluster.yaml?query=event%3Aschedule).

**Pull request merged**
